v3/docs/01-getting-started/e-glossary/index.mdx

```diff
@@ -7,640 +7,485 @@ category: getting started
+Adaptive quorum biasing allows for more flexible governance by removing the requirement to have an arbitrary quorum for voting purposes, which create undesirable governance mechanics.
+Adaptive quorum biasing is implemented in the [Democracy pallet](/v3/runtime/frame#democracy). 
+The Democracy pallet provides the interfaces for on-chain bodies such as a [collective](#council) or individual token holder—to call referenda with positive,
+negative, or neutral biases.
 
-A means of specifying a passing threshold for a referendum based on voter turnout. Positive turnout
-bias means that as more votes are cast, the passing threshold _decreases_; i.e. a higher turnout
-positively increases the likelihood of a referendum passing. Negative turnout bias means that the
-passing threshold _increases_ as more votes are cast. Negative turnout bias is also called "default
-carries" as, given an apathetic voting body, the referendum will pass by default. A neutral turnout
-bias specifies a simple majority passing threshold. AQB removes the need for strict quorums, which
-are arbitrary and create undesirable governance mechanics. AQB is implemented in the
-[Democracy pallet](/v3/runtime/frame#democracy), which provides the interfaces for a number of
-on-chain bodies (e.g. a [Collective](#council) or any token holder) to call referenda with positive,
-neutral, or negative biases.
+With a **positive turnout bias**, the passing threshold _decreases_ as more votes are cast, so that a higher turnout increases the likelihood of a referendum passing. 
+With a **negative turnout bias**, the passing threshold _increases_ as more votes are cast. 
+Negative turnout bias is also sometimes called a "default carries" position because if there's an apathetic voting body, the referendum passes by default. 
+A **neutral turnout bias** specifies a simple majority passing threshold.  
 
-## AfG
+<!--
+## AFG
 
-An internal codename for "Al's Finality Gadget", which is named after
-[Alistair Stewart](https://w3f-research.readthedocs.io/en/latest/team_members/alistair.html)
-who invented it. AfG is synonymous with [GRANDPA](#grandpa).
+An internal codename for "Al's Finality Gadget," which is named after [Alistair Stewart](https://w3f-research.readthedocs.io/en/latest/team_members/alistair.html)
+who invented it. 
+AFG is synonymous with [GRANDPA](#grandpa).
+-->
 
-## Aggregation
+## aggregation
 
-Used in the context of [FRAME](#frame), "[pallet](#pallet) aggregation" means combining analogous
-types from multiple runtime modules into a single type. This allows each module's analogous types to
-be represented. Currently there are six such data types:
+Used in the context of [FRAME](#frame), _aggregation_ or _[pallet](#pallet) aggregation_ is the process of combining analogous types from multiple runtime modules into a single type. 
+Pallet aggregation allows each module's analogous types to be represented. 
+Currently, there are six such data types:
```

Comment:
as a note, we also call this the "outer" object.

For example the "outer call" represents the aggregated call which is mentioned in this description. Maybe this is just nomenclature, but maybe also worth including.